Source/WebCore/html/HTMLAttachmentElement.h

```diff
@@ -101,12 +110,19 @@ class HTMLAttachmentElement final : public HTMLElement {
     bool childShouldCreateRenderer(const Node&) const final;
 #endif
 
+    enum class Implementation { Legacy, Modern, LegacyUnderModern };
```

Comment:
Not sure what LegacyUnderModern means.

Reply:
A legacy attachment (drawing the old way) that lives in the shadow tree under a modern attachment. Other suggestions?